code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -991,9 +991,9 @@
 		status_flags &= ~CANPUSH
 
 	if(module.clean_on_move)
-		flags_1 |= CLEAN_ON_MOVE_1
+		AddComponent(/datum/component/cleaning)
```

Comment:
Should use LoadComponent to preserve old behaviour

Reply:
...what old behaviour?

Reply:
if flags_1 were a list AddComponent would be += and LoadComponent would be |=

Reply:
so ```LoadComponent``` is ungodly slow and shouldn't be used?

Reply:
Not ungodly slow, but it should be avoided wherever possible. Its just a proc that does

```dm
. = GetComponent(...)
if(!.)
    return AddComponent(...)
```

Reply:
it was joke about it being the component equivalent of the ```|=``` list operator.